test/e2e/nightwatch.json

```diff
@@ -0,0 +1,63 @@
+{
+  "src_folders" : ["test/e2e/scenarios"],
+  "output_folder" : "reports",
+  "live_output": true,
+  "custom_commands_path" : "",
+  "custom_assertions_path" : "",
+  "page_objects_path" : "",
+  "globals_path" : "",
+  "test_workers" : {
+    "enabled" : true,
+    "workers" : "auto"
+  },
+
+  "test_runner" : {
+    "type" : "mocha",
+    "options" : {
+      "ui" : "bdd",
+      "reporter" : "list"
+    }
+  },
+
+  "selenium" : {
+    "start_process" : true,
+    "server_path" : "test/e2e/bin/selenium-server-standalone-3.4.0.jar",
```

Comment:
or https://www.npmjs.com/package/webdriver-manager for both selenium server and chromedriver bins. Probably preferred